docs/powerpoint/shapes.md

```diff
@@ -0,0 +1,129 @@
+
+The following code sample creates a straight line on the slide.
+
+```js
+// This sample creates a straight line on the first slide.
+PowerPoint.run(function (context) {
+    var shapes = context.presentation.slides.getItemAt(0).shapes;
+    var line = shapes.addLine(Excel.ConnectorType.straight, {left: 200, top: 50, height: 300, width: 150});
+    line.name = "StraightLine";
+    return context.sync();
+});
+```
+
+### Text boxes
+
+A text box is created with the [addTextBox](/javascript/api/powerpoint/powerpoint.shapecollection#addTextBox_text__options_) method. The first parameter is the text that should appear in the box initially. There is an optional second parameter of type [ShapeAddOptions](/javascript/api/powerpoint/powerpoint.shapeaddoptions) that can specify the initial size of the text box and its position relative to the top and left sides of the slide. Or these properties can be set after the shape is created.
+
+The following code sample shows how to create a text box on the first slide.
+
+```js
+// This sample creates a text box with the text "Hello!" and sizes it appropriately.
+PowerPoint.run(function (context) {
+    var shapes = context.presentation.slides.getItemAt(0).shapes;
+    var textbox = shapes.addTextBox("Hello!");
+    textbox.left = 100;
+    textbox.top = 100;
+    textbox.height = 300;
+    textbox.width = 450;
+    textbox.name = "Textbox";
+    return context.sync();
+});
+```
+
+## Move and resize shapes
+
+Shapes sit on top of the slide. Their placement is defined by the `left` and `top` properties. These act as margins from slide's respective edges, measured in points, with `left: 0` and `top 0` being the upper-left corner. The shape size is specified by the `height` and `width` properties. Your code can move or resize the shape by resetting these properties. (These properties have a slightly different meaning when the shape is a line. See [Lines](#lines).)
+
+## Text in shapes
+
+Geometric shapes can contain text. Shapes have a `textFrame` property of type [TextFrame](/javascript/api/powerpoint/powerpoint.textframe). The `TextFrame` object manages the text display options (such as margins and text overflow). `TextFrame.textRange` is a [TextRange](/javascript/api/powerpoint/powerpoint.textrange) object with the text content and font settings.
+
+The following code sample creates a geometric shape named "Braces" with the text "Shape text". It also adjusts the shape and text colors, as well as sets the text's vertical alignment to the center.
```

Comment:
One of the preceding samples shows the shape name in bold. I don't have a strong opinion about which way is better (bold or not bold), but I vote for consistency.
```suggestion
The following code sample creates a geometric shape named **"Braces"** with the text **"Shape text"**. It also adjusts the shape and text colors, as well as sets the text's vertical alignment to the center.
```